src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -361,66 +364,65 @@ static String validateConfig(Map<String, String> config) {
         || config
             .get(INGESTION_METHOD_OPT)
             .equalsIgnoreCase(IngestionMethodConfig.SNOWPIPE.toString())) {
-      if (!BufferThreshold.validateBufferThreshold(config, IngestionMethodConfig.SNOWPIPE)) {
-        configIsValid = false;
-      }
+      invalidConfigParams.addAll(BufferThreshold.validateBufferThreshold(config, IngestionMethodConfig.SNOWPIPE));
 
       if (config.containsKey(SnowflakeSinkConnectorConfig.ENABLE_SCHEMATIZATION_CONFIG)
           && Boolean.parseBoolean(
               config.get(SnowflakeSinkConnectorConfig.ENABLE_SCHEMATIZATION_CONFIG))) {
-        configIsValid = false;
         LOGGER.error(
             Logging.logMessage(
                 "Schematization is only available with {}.",
                 IngestionMethodConfig.SNOWPIPE_STREAMING.toString()));
+        invalidConfigParams.add(SnowflakeSinkConnectorConfig.ENABLE_SCHEMATIZATION_CONFIG);
+        // TODO @rcheng: question - can we add logging information in the error? this is confusing for customer
```

Comment:
would we want to add more info here? or should we rely on the customer self-debugging with their logs?